server/athenian/api/models/db/github/__init__.py

```diff
@@ -0,0 +1,223 @@
+from sqlalchemy import ARRAY, BigInteger, Boolean, Column, DateTime, Integer, String, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class VersionedMixin:
+    versions = Column("versions", ARRAY(Integer()), index=True)
+
+
+class IssueComment(Base, VersionedMixin):
+    __tablename__ = "github_issue_comments_versioned"
```

Comment:
Is this schema directly from the metadata-retrieval? Or it's something new?

Reply:
yeah, looks similar, but we have to find a way to easily update it

Reply:
Yes, I generated it with `sqlacodegen` from the real Postgres schema initialized by metadata-retrieval.